### PR TITLE
Export Prometheus flow metrics with UE IP labels

### PR DESCRIPTION
**Pull request by pudelkoM**

### CODE REVIEW

File: pfcpiface/bess.go [hunk 1]
```diff
@@ -671,6 +701,13 @@ func (b *bess) delPDR(ctx context.Context, done chan<- bool, p pdr) {
 			err error
 		)
 
+		// Only downlink PDRs contain the UE address.
+		if p.srcIP > 0 {
+			b.fseidMapMutex.Lock()
+			delete(b.fseidToIpMap, p.fseID)
```
Comment (krsna1729) [inline pfcpiface/bess.go:707]: tested with more than 1 downlink PDR?

Comment (krsna1729) [reply pfcpiface/bess.go:707]: and only one of them was deleted

File: pfcpiface/telemetry.go [hunk 2]
```diff
@@ -96,6 +97,12 @@ func (uc *upfCollector) Describe(ch chan<- *prometheus.Desc) {
 
 	ch <- uc.latency
 	ch <- uc.jitter
+
+	ch <- uc.sessionLatency
+	ch <- uc.sessionJitter
+	ch <- uc.sessionTxPackets
+	ch <- uc.sessionDroppedPackets
+	ch <- uc.sessionTxBytes
```
Comment (krsna1729) [inline pfcpiface/telemetry.go:105]: ideally we should have separate interface for these, like we have one for PFCP layer metrics. 

https://github.com/omec-project/upf-epc/blob/1ae479b109b9dfc62e341f0233af20d40e2c5d74/pfcpiface/metrics/interface.go#L48-L51

Overall split could be -

1. PFCP protocol metrics - interface - how many messages, sessions etc
2. PFCP fastpath metrics - interface - fastpath performance of the session/pdr, can benefit from more PFCP context
3. Fastpath backend specific metrics - need not be interface, registered/exported in their own files - fastpath wide latency. jitter, packets, bytes, dropped, memory util etc

**Review by osinstom — APPROVED** [review]
After discussion offline, I'm ok to merge it in this shape. We still would need to revise metrics APIs in future.

**Comment by pudelkoM** [comment]
retest this please

**Review by osinstom — APPROVED** [review]

**Pull request merged**